frontend/src/app/modules/work-space/work-space/work-space.component.ts

```diff
@@ -34,4 +35,12 @@ export class WorkSpaceComponent implements OnInit {
   logOut() {
     this.authService.logout();
   }
+
+  showNotifications() {
```

Comment:
Why not this.isShowNotifications = !this.isShowNotifications

Reply:
I didn't meet such statements before and I wasn't sure about how it works, so I decided to leave it clear